arangod/Aql/GraphNode.h

```diff
@@ -172,7 +172,8 @@ class GraphNode : public ExecutionNode {
   /// @brief the edge collection names
   std::vector<std::unique_ptr<aql::Collection>> _edgeColls;
 
-  /// @brief the vertex collection names
+  /// @brief the vertex collection names (can also be edge collections)
+  /// as an edge can also point to another edge, instead of a vertex.
```

Comment:
The closing parenthesis should be at the end of the second line:

```
  /// @brief the vertex collection names (can also be edge collections
  /// as an edge can also point to another edge, instead of a vertex).
```